src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -479,8 +479,8 @@ public void StopReceiving()
         /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
         /// <returns>Returns true on success</returns>
         /// <see href="https://core.telegram.org/bots/api#deletewebhook"/>
-        public Task<bool> DeleteWebhookAsync(CancellationToken cancellationToken = default)
-            => SendWebRequestAsync<bool>("deleteWebhook", null, cancellationToken);
+        public async Task<bool> DeleteWebhookAsync(CancellationToken cancellationToken = default)
```

Comment:
No need to return anything. If there are any errors, an exception would be thrown so return value(if successful) would always be `true` which is unnecessary to care for.